src/Elmish.WPF.Tests/BindingVmHelpersTests.fs

```diff
@@ -106,15 +115,19 @@ module Set =
 
 module Update =
 
-  let check<'a when 'a : equality> (g: Gen<'a>) =
-    Property.check <| property {
+  let check<'a when 'a: equality> (g: Gen<'a>) =
+    Property.check
+    <| property {
       let! initialModel = g
       let! newModel = g |> GenX.notEqualTo initialModel
 
       let model = ref initialModel
-      let dispatch msg = failwith $"Should not dispatch message {msg}"
-      let binding =
-        BindingData.TwoWay.id
+
+      let dispatch msg =
+        failwith $"Should not dispatch message {msg}"
```

Comment:
What is the setting for maximum line length?  This line seems plenty short and could stay on one line.

Reply:
Fixed.